ingest/ingest.go

```diff
@@ -84,6 +84,9 @@ func (i *FileIngest) getField(header string, query string, body string, c *fiber
 		}
 
 		bodyKey, err := root.GetKey(body)
+		if err != nil {
+			log.Println("Unable to get Key", err)
```

Comment:
Let's remove this, please. Server logs would become too cluttered if every user error was logged.